executor/aggfuncs/func_max_min.go

```diff
@@ -290,9 +311,202 @@ func (e *maxMin4Decimal) UpdatePartialResult(sctx sessionctx.Context, rowsInGrou
 			continue
 		}
 		cmp := input.Compare(&p.val)
-		if e.isMax && cmp == 1 || !e.isMax && cmp == -1 {
+		if e.isMax && cmp > 0 || !e.isMax && cmp < 0 {
```

Comment:
Why only change this line? line#363,411?

Reply:
caused by the implementation of different `compare` function.